ui/package.json

```diff
@@ -26,14 +27,15 @@
 	},
 	"devDependencies": {
 		"@rushstack/eslint-patch": "^1.2.0",
+		"@types/google.maps": "3.55.3",
 		"@types/marked": "^5.0.1",
 		"@vitejs/plugin-vue": "^4.2.1",
-		"volar-service-vetur": "latest",
 		"@vue/eslint-config-prettier": "^8.0.0",
 		"eslint": "^8.39.0",
 		"eslint-plugin-vue": "^9.11.0",
 		"prettier": "^3.0.1",
 		"vite": "^4.4.9",
-		"vite-plugin-monaco-editor": "^1.1.0"
+		"vite-plugin-monaco-editor": "^1.1.0",
+		"volar-service-vetur": "latest"
```

Comment:
Unrelated but since you pointed out the issue with `remixicon`, can you tell me whether this `latest` is valid?